crates/tauri-cli/src/build.rs

```diff
@@ -131,6 +137,32 @@ pub fn setup(
   mobile: bool,
 ) -> Result<()> {
   let tauri_path = tauri_dir();
+
+  log::info!("Looking up installed plugins to check incompatible versions...");
```

Comment:
I'm still a bit concerned about this blocking the build process, is it possible that we do this in another thread as well?

This is the time I got in the `plugin-workspace`'s API example

```
        Info Looking up installed plugins to check incompatible versions...
[crates\tauri-cli\src\build.rs:166:3] start.elapsed() = 1.4956269s
```